src/App.ts

```diff
@@ -169,10 +160,10 @@ export default class App {
   private logger: Logger;
 
   /** Authorize */
-  private authorize!: Authorize;
+  private authorize!: Authorize<false>;
 
   /** Org Authorize */
-  private orgAuthorize!: Authorize;
```

Comment:
This was technically wrong previously. The `Authorize` type describes a function which non-optionally took a `teamId`.